.circleci/config.yml

```diff
@@ -1,4 +1,4 @@
-version: 2.0
+version: 2.1
```

Comment:
I think we ended up eating the `composer test-integration` part (different from `composer test-integrations-XX`)

Reply:
Its there but its not immediately obvious. I remove the yaml reference in lieu of normal `run` devfinition to make it more obvious